src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponentSpec.scala

```diff
@@ -0,0 +1,150 @@
+
+    val methodConfig2: MethodConfiguration = MethodConfiguration(
+      "test_namespace",
+      "test_name2",
+      "sample",
+      Map("prereq_2" -> AttributeString("value"), "prereq_2.1" -> AttributeString("value2")),
+      Map("input_2" -> AttributeString("value"), "input_2.1" -> AttributeString("value2")),
+      Map("output_2" -> AttributeString("value"), "output_2.1" -> AttributeString("value2")),
+      MethodRepoMethod("repo_namespace", "repo_name", 2)
+    )
+
+    val methodConfig3: MethodConfiguration = MethodConfiguration(
+      "test_namespace",
+      "test_name3",
+      "sample",
+      Map("prereq_3" -> AttributeString("value3")),
+      Map("input_3" -> AttributeString("value3")),
+      Map("output_3" -> AttributeString("value3")),
+      MethodRepoMethod("repo_namespace", "repo_name", 3)
+    )
+
+    val updatedMethodConfig1: MethodConfiguration =  MethodConfiguration(
+      "test_namespace",
+      "test_name",
+      "pair",
+      Map("prereq_NEW" -> AttributeString("value")),
+      Map("input_NEW" -> AttributeString("value")),
+      Map("output_NEW" -> AttributeString("value")),
+      MethodRepoMethod("repo_namespace", "repo_name", 1)
+    )
+
+    //save config #1
+    assertResult(methodConfig) {
+      runAndWait(methodConfigurationQuery.save(workspaceId, methodConfig))
+    }
+
+    //save config #2
+    assertResult(methodConfig2) {
+      runAndWait(methodConfigurationQuery.save(workspaceId, methodConfig2))
+    }
+
+    //save config #3
+    assertResult(methodConfig3) {
+      runAndWait(methodConfigurationQuery.save(workspaceId, methodConfig3))
+    }
+
+    //load config #1
+    assertResult(Option(methodConfig)) {
+      runAndWait(methodConfigurationQuery.get(workspaceId, methodConfig.namespace, methodConfig.name))
+    }
+
+    //load config #2
+    assertResult(Option(methodConfig2)) {
+      runAndWait(methodConfigurationQuery.get(workspaceId, methodConfig2.namespace, methodConfig2.name))
+    }
+
+    //list method configs in a workspace
+    assertResult(Seq(methodConfig.toShort, methodConfig2.toShort, methodConfig3.toShort)) {
+      runAndWait(methodConfigurationQuery.list(workspaceId))
+    }
+
+    //rename config #3
+    assertResult(1) {
+      runAndWait(methodConfigurationQuery.rename(workspaceId, methodConfig3.namespace, methodConfig3.name, "updatedName"))
+    }
+
+    //load config #3 with new name
+    assertResult(Option(methodConfig3.copy(name = "updatedName"))) {
+      runAndWait(methodConfigurationQuery.get(workspaceId, methodConfig3.namespace, "updatedName"))
+    }
+
+    //re-save config #1 with different inputs/outputs/prereqs
+    assertResult(updatedMethodConfig1) {
+      runAndWait(methodConfigurationQuery.save(workspaceId, updatedMethodConfig1))
+    }
+
+    //load the updated version of config #1
+    assertResult(Option(updatedMethodConfig1)) {
+      runAndWait(methodConfigurationQuery.get(workspaceId, updatedMethodConfig1.namespace, updatedMethodConfig1.name))
```

Comment:
fair enough, but these tests are going to be replaced by the ported tests from GraphMethodConfigDAOSpec